media/js/firefox/australis/fx36/whatsnew.js

```diff
@@ -0,0 +1,53 @@
+;(function($, Mozilla) {
+    'use strict';
+
+    var $tourStrings = $('#strings');
+    var helloAnimationStage = $('#hello-animation-stage');
+
+    //track if this is the first time a user has seen tour
+    function trackFirstTimeUse() {
+        var firstTime = 'True';
+        try {
+            if (localStorage.getItem('mozUITourGlobalFlag') === 'taken') {
+                firstTime = 'False';
+            } else {
+                localStorage.setItem('mozUITourGlobalFlag', 'taken');
+            }
+            gaTrack(['_trackEvent', 'Tour Interaction', 'First Time Seeing Tour', firstTime, 0, true]);
```

Comment:
Hmm, can't see this event coming through in GA. 

I see it being passed in the console:

`utme	5(Tour Interaction*First Time Seeing Tour*False)(0)8(12!Hello Available)9(12!Yes)11(12!2)`

But can't confirm it in GA Real Time.

Its equivalent is appearing in `firstrun.js`. The only difference is that it's not trying to pass a custom dimension at the same time. I wonder if that's mucking things up for some reason.

Ever seen anything like this?